internal/orchestrion/_.orchestrion.yml

```diff
@@ -0,0 +1,48 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
why does this file start with `_`?

Reply:
This file isn't a standard `orchestrion.yml`; in the sense that it MUST be loaded iif the tracer is auto-injected...

That said, I renamed it to `gls.orchestrion.yml` instead, as it looks less weird.